kong/clustering.lua

```diff
@@ -942,34 +942,47 @@ function _M.init_worker(conf)
     local push_config_semaphore = semaphore.new()
 
     -- Sends "clustering", "push_config" to all workers in the same node, including self
-    local function post_push_config_event_to_node_workers(data)
-      if type(data) == "table" and data.schema and
-         data.schema.db_export == false
-      then
+    local function post_push_config_event()
+      local res, err = kong.worker_events.post("clustering", "push_config")
+      if not res then
+        ngx_log(ngx_ERR, "unable to broadcast event: ", err)
+      end
+    end
+
+    -- Handles "clustering:invalidation" cluster event
+    local function handle_clustering_invalidation_event(data)
+      ngx_log(ngx_DEBUG, "received clustering:invalidation event for ", data)
+      post_push_config_event()
+    end
+
+
+    -- Handles "dao:crud" worker event and broadcasts "clustering:invalidation" cluster event
+    local function handle_dao_crud_event(data)
+      if type(data) ~= "table" or data.schema == nil or data.schema.db_export == false then
         return
       end
 
+      kong.cluster_events:broadcast("clustering:invalidation", data.schema.name .. ":" .. data.operation)
```

Comment:
Since this event does not come from or are driven by the DAO subsystem anymore, maybe `clustering:push_config` makes this difference more obvious?